Formula/imagemagick.rb

```diff
@@ -69,6 +71,9 @@ def install
       --enable-static
     ]
 
+    args << (build.without? "fontconfig") ? "--without-fontconfig" : "--with-fontconfig"
+    args << (build.without? "freetype") ? "--without-freetype" : "--with-freetype"
+
     if build.without? "modules"
```

Comment:
But then it won't be enabled by default. I tried building with the default configure options on a machine that has fontconfig and it won't build with fontconfig support.